src/shisen_cpp/camera/node/camera_node.cpp

```diff
@@ -31,9 +33,7 @@ CameraNode::CameraNode(rclcpp::Node::SharedPtr node, const Options & options)
 {
 }
 
-CameraNode::~CameraNode()
-{
-}
+CameraNode::~CameraNode() {}
```

Comment:
it seems like your prettier or format is different with our ROS format, so please check it because there are so many line changes